src/main/java/training/commands/Roller/stopRollerCommand.java

```diff
@@ -0,0 +1,30 @@
+package training.commands.Roller;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.Roller.Roller;
+
+public class stopRollerCommand extends Command {
+    private final Roller roller = Roller.getInstance();
+
+    public stopRollerCommand() {
+        addRequirements(roller);
+    }
+
+    @Override
+    public void execute() {
+        addRequirements(roller);
+        roller.stop();
+    }
+
+    @Override
+    public boolean isFinished() {
+        try {
+            wait(5);
+        } catch (InterruptedException e) {
+            SmartDashboard.putString("got error on stopRollerCommand InterruptedException: ", e.getMessage());
+        }
+        return true;
```

Comment:
no, when you do wait it stuck the whole code and we dont want it

Reply:
I'm worried that it'll immediately stop before applying the changes. How can I fix this